kube-api-audit-policy

```diff
@@ -1,71 +1,153 @@
-# Kubernetes audit policy - this is a sample audit policy, modify to suit your
```

Comment:
What is this file now? Just wondering if we can provide some more context to people - there are some changes made from the file that existed previously but I'm not clear exactly where they came from.

Reply:
This file is the one that you send me, it's the configuration from the GKE github. 

https://github.com/kubernetes/kubernetes/blob/08ddda629e906633ffdba6ea4cd9ebf2cc21dd23/cluster/gce/gci/configure-helper.sh#L706